src/osdep/amiberry_gfx.cpp

```diff
@@ -657,6 +657,11 @@ int sdl2_render_thread(void *ptr) {
 void show_screen(int mode)
 {
 	const auto start = read_processor_time();
+
+#ifdef USE_SDL2
+	// Alynna // This hack is to keep good timing in RTG.
+	currprefs.gfx_framerate = screen_is_picasso;
+#endif
```

Comment:
Could you explain the logic behind this please?
screen_is_picasso should return a boolean, but gfx_framerate is an int...